### PR TITLE
Add Chanining in AlertController

### PR DESCRIPTION
**Pull request by Stjng**

### CODE REVIEW

File: SwiftUtils/Classes/AlertController.swift [hunk 1]
```diff
@@ -18,11 +18,20 @@ public enum AlertLevel: Int {
 public class AlertController: UIAlertController {
     public var level = AlertLevel.Normal
 
-    public func addAction(title: String?, style: UIAlertActionStyle = UIAlertActionStyle.Default, handler: (() -> Void)? = nil) {
+    public static func create(title: String, message: String, level: AlertLevel = .Normal, preferredStyle: UIAlertControllerStyle = .Alert) -> AlertController {
```
Comment (zendobk) [inline SwiftUtils/Classes/AlertController.swift:21]: please update `init` function instead of creating new one

Comment (Stjng) [reply SwiftUtils/Classes/AlertController.swift:21]: How to return itself in `init` function.

Comment (zendobk) [reply SwiftUtils/Classes/AlertController.swift:21]: what do you means? `init` function returns an instance as default, right?

Comment (Stjng) [reply SwiftUtils/Classes/AlertController.swift:21]: According to Apple 

> The UIAlertController class is intended to be used as-is and does not support subclassing. The view hierarchy for this class is private and must not be modified.

So when i `convenience init` can cause app crash.

Comment (zendobk) [reply SwiftUtils/Classes/AlertController.swift:21]: so, simply is remove this function

Comment (Stjng) [reply SwiftUtils/Classes/AlertController.swift:21]: Why we cannot use static function like an extension :(

**Comment by zendobk** [comment]
@Stjng This branch is out-of-date with the base branch. Please update your repo first.

File: SwiftUtils/Classes/AlertController.swift (hunk 1 above)
Comment (zendobk) [reply SwiftUtils/Classes/AlertController.swift:21]: default `init` is good, so there's no reason to write new one

**Pull request closed without merging**